clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -115,8 +120,8 @@ public abstract CreateTopicsResult createTopics(Collection<NewTopic> newTopics,
                                                     CreateTopicsOptions options);
 
     /**
-     * Similar to #{@link AdminClient#deleteTopics(Collection<String>, DeleteTopicsOptions)},
-     * but uses the default options.
+     * Similar to #{@link AdminClient#deleteTopics(Collection, DeleteTopicsOptions)},
+     * but uses the default options. See {@link #deleteTopics(Collection, DeleteTopicsOptions)} for more details.
```

Comment:
Hi, may I ask why do you do @link instead of @see annotations? :)

Reply:
@viktorsomogyi Thanks for the comment. I am not aware of the difference between `@link` and `@see`. I used `@link` because it is currently used in `AdminClient`. Let me look into the difference between the two and update the pull request accordingly.

Reply:
@viktorsomogyi I just checked the difference between `@link` and `@see`. `@link` simply creates a link to another method in AdminClient. `@see` is a bit more intensive because it creates a specific section named `see also` in the javadoc. I don't have a strong opinion which one should be used here. For the purpose of this patch I would keep the existing javadoc format and focus on improving the javadoc content.

Thanks for the comment.